src/database/migrations/0024-alter-bulk_uploads.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE bulk_uploads
+  ADD COLUMN created_by INTEGER NOT NULL DEFAULT select_system_user_id(),
```

Comment:
Also flagging this: I don't want a default for this column and I also don't think we want to allow for bulk uploads with no user.  For any existing data in the system the bulk uploads will be attributed to the system user (and then you see that default is immediately dropped)

Reply:
Agreed, I don't think there's a scenario where we want to allow non-authenticated users to upload data. Just covering any existing bulkuploads for migrations sake seems sufficient